### PR TITLE
Emoji in ouija board word choices

### PR DESCRIPTION
**Pull request by pali6**
## About the PR 
This PR adds a 30% chance for there to be 1-3 emoji in the word choices for an ouija board. Sadly due to byond limitations these emoji are tiny and black and white in the choice list so I also changed it to tgui input instead. (And in order to make that work I added an optional argument which allows you to have illegal characters in buttons and also changed the css a bit.)
image

Now also adds a \*REFRESH\* button which gives you a new selection of words.
Also an admin spawnable Emouija board which contains only emoji.

Let me know if the 30% chance should be higher or lower in your opinion.

## Why's this needed? 
I think it'd be funny and potentially interesting to communicate via emoji. But I'm also not sure about the general immersion-breaking and stuff of this feature so let me know what you think.

## Changelog
```changelog
(u)pali
(*)Ouija boards have moved to the 21st centure and now contain emoji
```

### CODE REVIEW

File: code/obj/item/device/audio_recorder.dm [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	mats = null
+
+	var/recording = FALSE
+	var/list
+	var/action_id = 0
+
+	New()
+		..()
+
+	proc/start_playing()
```
Comment (ZeWaka) [inline code/obj/item/device/audio_recorder.dm:21]: umm pali i don't think you meant to commit this

File: code/modules/tgui/tgui_input_list.dm [hunk 2]
```diff
@@ -45,7 +46,7 @@
  * * callback - The callback to be invoked when a choice is made.
  * * timeout - The timeout of the input box, after which the menu will close and qdel itself. Set to zero for no timeout.
  */
-/proc/tgui_input_list_async(mob/user, message, title, list/buttons, datum/callback/callback, timeout = 60 SECONDS)
+/proc/tgui_input_list_async(mob/user, message, title, list/buttons, datum/callback/callback, timeout = 60 SECONDS, allowIllegal = FALSE)
```
Comment (ZeWaka) [inline code/modules/tgui/tgui_input_list.dm:49]: could you doc allowillegal above

Comment (pali6) [reply code/modules/tgui/tgui_input_list.dm:49]: Whoops, totally forgot about that one.

**Comment by vocalpocal** [comment]
oumoji cool

**Pull request merged**